metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
+
+// regex match github.com/([^/]+/[^/]+) on reply of https://gopkg/?go-get=1
+func findGitHubRepo(gopkg string) (string, error) {
+	if strings.HasPrefix(gopkg, "github.com/") {
+		return gopkg[len("github.com/"):], nil
+	}
+	resp, err := http.Get("https://" + gopkg + "?go-get=1")
+	if err != nil {
+		return "", err
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	resp.Body.Close()
+	if err != nil {
+		return "", err
+	}
+	repoMatch := githubRegexp.FindSubmatch(body)
```

Comment:
well, like gopkg.in,
```
<meta name="go-import" content="gopkg.in/go-playground/validator.v8 git https://gopkg.in/go-playground/validator.v8">
<meta name="go-source" content="gopkg.in/go-playground/validator.v8 _ https://github.com/go-playground/validator/tree/v8.18.2{/dir} https://github.com/go-playground/validator/blob/v8.18.2{/dir}/{file}#L{line}">
```
it only show the github repo in go-source meta. And we still need to do a regexp match for the content of go-source.

But go-source is not always shown. like pault.ag,

```
                <meta charset="utf-8">
                <title>pault.ag/go/config</title>
                <meta name="go-import" content="pault.ag/go/config git https://github.com/paultag/go-config">
        </head>
```
paultag only uses go-import :)

IMHO, regexp is much simpler.